LibreHardwareMonitor/UI/MainForm.Designer.cs

```diff
@@ -139,7 +139,6 @@ private void InitializeComponent()
             this.splitContainer = new LibreHardwareMonitor.UI.SplitContainerAdv();
             this.treeView = new Aga.Controls.Tree.TreeViewAdv();
             this.batteryMenuItem = new System.Windows.Forms.ToolStripMenuItem();
-            this.psuMenuItem = new System.Windows.Forms.ToolStripMenuItem();
```

Comment:
Removed this line as it was being double declared and preventing Designer from loading the Form.